posthog/models/integration.py

```diff
@@ -1098,8 +1098,45 @@ def refresh_access_token(self):
             oauth_refresh_counter.labels(self.integration.kind, "success").inc()
         self.integration.save()
 
-    def create_issue(self, team_id: str, posthog_issue_id: str, config: dict[str, str]):
-        pass
+    def organization(self) -> str:
+        return dot_get(self.integration.config, "account.name")
+
+    def list_repositories(self, page: int = 1) -> list[dict]:
+        org = self.organization()
+        access_token = self.integration.sensitive_config["access_token"]
+
+        response = requests.get(
+            f"https://api.github.com/orgs/{org}/repos?sort=pushed&order=desc&page={page}",
```

Comment:
This will just get the first page for now, ordered by most recently pushed to. GitHub API doesn't offer any type of search for this endpoint so I'm not going to worry about pagination until we get a request for it